sdk/src/main/java/com/endare/adhese/sdk/AdheseOptions.java

```diff
@@ -59,6 +64,15 @@ public String getAsURL() {
         }
 
         builder.append(String.format("/%s%s", AdheseParameter.COOKIE_MODE.getKey(), cookieMode.getValue()));
+        TreeMap<String, Set<String>> sorted = new TreeMap<>(customParameters);
+        for (Map.Entry<String, Set<String>> customParameter: sorted.entrySet()) {
+            StringBuilder values = new StringBuilder();
+            for (String value: customParameter.getValue()) {
```

Comment:
these also need sorting to have deterministic tests

Reply:
how do you mean? what would you test that eg `custom_parameter_case_list_multiple` doesn't cover?